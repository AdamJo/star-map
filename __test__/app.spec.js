import { h, Component } from 'preact';
import render from 'preact-render-to-string';
import App from '../src/views/';

describe('App', () => {
  it('exports `App`', () => {
    expect(typeof App).toBe('function');
  });
});

test('snapshot is working!', () => {
  const app = render(App);
  expect(app).toMatchSnapshot();
});
