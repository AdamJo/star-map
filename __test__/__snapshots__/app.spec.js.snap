// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot is working! 1`] = `"<div id=\\"app\\"><header class=\\"header\\"><h1>Sup</h1><nav></nav></header><main id=\\"content\\"><div class=\\"page page__404\\"><div class=\\"card\\"><h1>404 Page</h1><p>Looks like you were given a bad link ;-)</p><pre>blank</pre></div></div></main></div>"`;
